scripts/e2e-tests.sh

```diff
@@ -93,15 +93,30 @@ function dump_cluster_state() {
   echo "***         E2E TEST FAILED         ***"
   echo "***    Start of information dump    ***"
   echo "***************************************"
-  echo ">>> All resources:"
-  kubectl get all --all-namespaces --show-labels -o wide
-  echo ">>> Services:"
-  kubectl get services --all-namespaces --show-labels -o wide
-  echo ">>> Events:"
-  kubectl get events --all-namespaces --show-labels -o wide
-  function_exists dump_extra_cluster_state && dump_extra_cluster_state
-  echo ">>> Full dump: ${ARTIFACTS}/k8s.dump.txt"
-  kubectl get all --all-namespaces -o yaml > ${ARTIFACTS}/k8s.dump.txt
+
+  local output="${ARTIFACTS}/tmp/k8s.dump.txt"
+  echo ">>> The dump is located at ${output}"
+
+  for crd in `kubectl api-resources --verbs=list -o name | sort`; do
```

Comment:
$() is preferred over `` for readability sake.

```suggestion
  for crd in $(kubectl api-resources --verbs=list -o name | sort); do
```